addons/point_of_sale/models/pos_order.py

```diff
@@ -669,10 +681,10 @@ def action_receipt_to_customer(self, name, client, ticket, order_ids=False):
                 'datas': base64.b64encode(report[0]),
                 'store_fname': filename,
                 'res_model': 'account.move',
-                'res_id': order_ids[0],
+                'res_id': orders.account_move.id,
```

Comment:
hum I don't know if we want to link it to the `account.move` or to the `pos.order`. I'd say the `pos.order`